crates/consensus/src/transaction/typed.rs

```diff
@@ -307,6 +336,61 @@ impl Transaction for OpTypedTransaction {
     }
 }
 
+impl SignableTransaction<Signature> for OpTypedTransaction {
```

Comment:
hmm, we don't have this impl for typedtx on alloy,

need to figure out if we actually want this